### DISCUSSION TITLE
Stop live transcription should stop capturing audio, but finish processing?

### DISCUSSION THREAD

**Discussion by cenzovit** [mybigday/whisper.rn#51] [Ideas]
Maybe I have something configured incorrectly; however, when doing realtime transcription, calling stop aborts the transcription and an @RNWhisper_onRealtimeTranscribeEnd event gets fired with no listener. If it is possible, it seems more practical for there to be two methods, stop as it is currently implemented, and stopCapturing which could stop capturing audio input but finish transcribing? This is most noticeable on dev builds since the transcription isn't quite realtime and falls behind.

**Comment by jhen0409** [comment]
It definitely reasonable to add more flexible API like stopCapturing, stopTranscribing.

Also, we may need to wait last transcribe event to fix the problem. https://github.com/mybigday/whisper.rn/blob/76c358a50b5c7da9ee5aaf8a0bc5c8d89b1203de/src/index.ts#L241
